src/tests/build.cmd

```diff
@@ -130,7 +150,40 @@ if [!processedArgs!]==[] (
     )
 )
```

Comment:
This logic was unexpectedly _above_ the `:ArgsDone` label and was being hit whenever there was an unrecognized argument (vs. when we were done processing arguments).

Reply:
This was working by coincidence that argument parsing aborts upon the first unrecognized argument. There are existing pipelines relying on that behavior.

Nonetheless, the logic is moved below the label now, even though we still exit processing upon the first unrecognized argument.